core/src/main/java/com/linecorp/armeria/common/RequestContext.java

```diff
@@ -167,6 +168,11 @@
     @Nullable
     SSLSession sslSession();
 
+    /**
+     * Returns the {@link UUID} that contains the information about the current {@link Request}.
```

Comment:
`Returns the {@link UUID} that represents the unique identifier of ....` (See @minwoox's comment)